src/test/java/org/exmple/utility/UtilityMethodsTest.java

```diff
@@ -175,4 +177,26 @@ public void countNegative_shouldReturnException() {
         int[] array = null;
         assertThrows(NullPointerException.class, () -> UtilityMethods.countNegative(array));
     }
+
+    @Test
+    public void compareByNegativeCount_shouldReturnOne() {
```

Comment:
There should be more test cases:
1. One test case when a < b
2. One test case when a == b
3. ~One test case when a > b (the current test describes this case)~

Please add 1 and 2 test cases